.github/workflows/tests.yml

```diff
@@ -19,37 +23,39 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ["2.12", "2.17"]
+        version: ["2.12.4", "2.19.0"]
     name: integration-tests (dart ${{ matrix.version }})
-    container:
-      image: dart:${{ matrix.version }}
+
     services:
       meilisearch:
         image: getmeili/meilisearch:latest
         env:
           MEILI_MASTER_KEY: "masterKey"
           MEILI_NO_ANALYTICS: "true"
         ports:
-          - "7700:7700"
+          - 7700:7700
+
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+      - uses: dart-lang/setup-dart@v1
+        with:
+          sdk: ${{ matrix.version }}
       - name: Install dependencies
         run: dart pub get
       - name: Run integration tests
-        run: dart run test
-        env:
-          MEILISEARCH_URL: http://meilisearch:7700
+        run: dart test --exclude-tags=browser --concurrency=1
+      #  TODO: Add Chrome to this CI action
+      # - name: Run Chrome tests
+      #   run: dart run test --platform=chrome --tags=browser
 
```

Comment:
Maybe it's worth to create an issue about it to not forget about it

Reply:
Yep, it is!